meeting/urls.py

```diff
@@ -10,5 +10,6 @@
     path('projects/<int:project_key>', views.projects_view, name='projects_view'),
     path('projects/<int:project_key>/edit', views.ProjectModificationProcess.as_view(), name="ProjectModificationProcess"),
     path('login', views.LoginProcess.as_view(), name="LoginProcess"),
-    path('register', views.RegisterProcess.as_view(), name="RegisterProcess")
+    path('register', views.RegisterProcess.as_view(), name="RegisterProcess"),
+    path('create_meetings', views.createMeetings, name='meetings')
```

Comment:
Some of our URL routes are going to be different, I think, due to the different styles we have. We might run into issues with regard to cohesiveness later when performing heavy-duty integrations with each others' work, but for now (and especially since we neglected to mention the enforcement of route styles before starting), this will work great for now.